config/puma/production.rb

```diff
@@ -17,3 +25,6 @@
   dynflow = ::Rails.application.dynflow
   dynflow.initialize! unless dynflow.config.lazy_initialization
 end
+
+# === Puma control rack application ===
+activate_control_app 'unix:///var/run/foreman/foreman-pumactl.sock'
```

Comment:
puma automatically creates auth token, so we don't need to explicitly supply it.